### PR TITLE
Tags parameter for redirects

### PR DESCRIPTION
**Pull request by philipp-r**
see issue #833

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1044,7 +1044,7 @@ function renderPage($conf, $pluginManager, $LINKSDB)
         // Show login screen, then redirect to ?post=...
         if (isset($_GET['post']))
         {
-            header('Location: ?do=login&post='.urlencode($_GET['post']).(!empty($_GET['title'])?'&title='.urlencode($_GET['title']):'').(!empty($_GET['description'])?'&description='.urlencode($_GET['description']):'').(!empty($_GET['source'])?'&source='.urlencode($_GET['source']):'')); // Redirect to login page, then back to post link.
+            header('Location: ?do=login&post='.urlencode($_GET['post']).(!empty($_GET['title'])?'&title='.urlencode($_GET['title']):'').(!empty($_GET['description'])?'&description='.urlencode($_GET['description']):'').(!empty($_GET['tags'])?'&tags='.urlencode($_GET['tags']):'').(!empty($_GET['source'])?'&source='.urlencode($_GET['source']):'')); // Redirect to login page, then back to post link.
```
Comment (ArthurHoaro) [inline index.php]: While you're at it, could you split this line to make it readable?

Comment (philipp-r) [inline index.php]: done 👌

Comment (ArthurHoaro) [inline index.php]: 👍 
If you could squash your commits into one it would be perfect.

Comment (philipp-r) [inline index.php]: Okay. I hope it is ready to be merged now. 😉

**Pull request merged**